integration/create_topic_test.go

```diff
@@ -0,0 +1,75 @@
+package integration_test
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/satori/go.uuid"
+)
+
+var _ = Describe("Create Topic", func() {
+	Context("When topic already exists", func() {
+		var topicName string
+		var response *http.Response
+		BeforeEach(func() {
+			topicName = uuid.NewV4().String()
+			var err error
+			response, err = http.Post("http://localhost:12345/"+topicName, "text/plain", nil)
+			Expect(err).ToNot(HaveOccurred())
+			response.Body.Close()
+			response, err = http.Post("http://localhost:12345/"+topicName, "text/plain", nil)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		AfterEach(func() {
+			response.Body.Close()
```

Comment:
tbh. This close failing won't damage anything hence didn't bother with checking the error ...